src/main/java/com/spotify/github/v3/clients/GitHubClient.java

```diff
@@ -565,6 +600,15 @@ <T> CompletableFuture<T> post(final String path, final String data, final Class<
             response -> json().fromJsonUncheckedNotNull(responseBodyUnchecked(response), clazz));
   }
 
+  CompletableFuture<Response> postGraphql(final String data) {
```

Comment:
GraphQL is always post IIRC
```suggestion
  CompletableFuture<Response> graphql(final String data) {
```